recipes/libavif/all/conanfile.py

```diff
@@ -2,6 +2,7 @@
 import os
 import textwrap
 
+# pylint: skip-file
```

Comment:
Not sure than defeating hooks is allowed.

I've submitted https://github.com/conan-io/conan-center-index/pull/13007

Reply:
@SpaceIm thank you! I was thinking about it today!